helpers/image_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'tempfile'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    Tempfile.create('images-', StaticData::TMP_FOLDER.to_s) do |fo|
+      fo.binmode
+      fo.write(URI.parse(url).open(&:read))
+      fo.rewind
+      File.size(fo.path)
+    end
+  end
+end
```

Comment:
rubocop

Reply:
I use "amend" after review

Reply:
It would be great to make one more commit to change the code style. Also, it's a bad idea to make `amend` because it can  break some of  the reviewed code. In this case, it will be necessary to review the code once again.